### PR TITLE
Version 4.1

### PR DESCRIPTION
**Pull request by jongund**
* Adds escape key to hide tooltip when button has focus
* Adds configuration options for `font-family` and `font-size`, otherwise as before they are inherited from the page
* Fixes some CSS selector issues of colors and size not being inherited from a menu item styling to `.label` class

### CODE REVIEW

**Comment by brianteeman** [comment]
I will check this over the weekend.

**Comment by brianteeman** [comment]
Can you explain why there is a font option? I can't see the benefit for this as it can already be styled using the site css

**Comment by jongund** [comment]
> Can you explain why there is a font option? I can't see the benefit for this as it can already be styled using the site css

In some web development environments it is easier to use the configuration object, rather than adding a style sheet to change default rendering.  For example WordPress and Drupal environments.    We are currently working with someone at Illinois developing a WordPress plugin for SkipTo.

**Comment by brianteeman** [comment]
Might be easier but it would be the wrong way to do it imho. But maybe thats just the purist in me.

Still it doesn't explain why you are now setting a font family for the aria style.

File: README.md [hunk 1]
```diff
@@ -40,6 +44,16 @@ NOTE: Browsers on iOS and iPadOS devices support `accesskey`s to move focus to t
...
 
+## Access keys
+
+Access keys work  just like regular shortcut keys except that they need a browser-specific modifier key in order to work. For example, to use the "SkipTo" access key, you would press the modifier key + the access key ("0" is the default accesskey). A list for how this would work in most popular browsers and operating systems.
+
+* Mozilla Firefox -- Alt+Shift+0 (Windows or Linux) and Control+Option+0 (Mac OS)
+* Google Chrome and Opera -- Alt+0 (Windows or Linux) and Control+Option+0 (Mac OS).
+* Safari -- Control+Option+0 (MacOS).
+
+NOTE: Browsers on iOS and iPadOS devices support `accesskey`s to move focus to the menu button, but to not support the menu button keyboard commands to use the menu at this time.  Browsers on Android devices do not support `accesskey`s at this time.
```
Comment (brianteeman) [inline README.md:55]:
```suggestion
NOTE: Browsers on iOS and iPadOS devices support `accesskey`s to move focus to the menu button, but do not support the menu button keyboard commands to use the menu at this time.  Browsers on Android devices do not support `accesskey`s at this time.
```

typo to ===>do

File: README.md [hunk 2]
```diff
@@ -384,6 +406,12 @@ Happy skipping!
 
 ## Version History
 
+### Version 4.1
+* Added feature for the <kbd>escape</kbd> key to hide tooltip when focus is on button.
+* Added new properties to set font family and font size.
+* Adding CSS properties to the <code>.label</code> and <code>.level</code> class so the inherited values from <code>[role="menuitem"]</code> are not overridden as easily by other stylesheets used on a page.
+* Updated moving focus to improve moving focus to visible targets with in landmarks.
```
Comment (brianteeman) [inline README.md:413]:
```suggestion
* Updated moving focus to improve moving focus to visible targets within landmarks.
```

typo

**Comment by jongund** [comment]
> Might be easier but it would be the wrong way to do it imho. But maybe thats just the purist in me.
> 
> Still it doesn't explain why you are now setting a font family for the aria style.

I needed the font-family for a change to one of the ARIA authoring practice examples for landmarks:
https://raw.githack.com/w3c/aria-practices/update-landmarks/examples/landmarks/index.html

File: README.md (hunk 1 above)
Comment (jongund) [reply README.md:55]: fixed, thanks for the edits

File: README.md (hunk 2 above)
Comment (jongund) [reply README.md]: fixed, thanks for the edits

**Comment by brianteeman** [comment]
Sorry for the delay. I will test this again first thing friday morning. Assuming I find no problems I will merge it and publish a release **unless** you want me to wait for anything else first?

**Comment by jongund** [comment]
No nothing else at this time, thank you for reviewing.

**Comment by brianteeman** [comment]
OK great I will get on to that in the morning. 11pm here

**Comment by brianteeman** [comment]
If you are OK with it I would like to merge #92 which just updates the readme to reference Joomla and to add myself to the list of contributors.

**Comment by brianteeman** [comment]
4.1.0 package released to npm

**Comment by jongund** [comment]
@brianteeman 
How does the `SkipTo` npm package work?
Should we have some documentation on the `README.md` page?

**Comment by brianteeman** [comment]
Yes probably - guess I will put that on my todo list

**Comment by jongund** [comment]
Thanks, some people have been asking about npm package and I would like to learn more about it.

**Comment by brianteeman** [comment]
Google is probably your best explainer. Basically if you use a lot of external scripts then you can keep them all upto date with one command

**Comment by brianteeman** [comment]
This is the best explainer I have seen https://qr.ae/pGKTlG

**Pull request merged**